website/docs/r/extension_servicenow.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_extension_servicenow"
+sidebar_current: "docs-pagerduty-resource-extension-servicenow"
+description: |-
+  Creates and manages a ServiceNow service extension in PagerDuty.
+---
+
+# pagerduty\_extension\_servicenow
+
+A special case for [extension](https://v2.developer.pagerduty.com/v2/page/api-reference#!/Extensions/post_extensions) for ServiceNow.
+
+## Example Usage
+
+```hcl
+data "pagerduty_extension_schema" "webhook" {
+  name = "Generic V2 Webhook"
+}
+
+resource "pagerduty_user" "example" {
+  name  = "Howard James"
+  email = "howard.james@example.domain"
+  teams = [pagerduty_team.example.id]
+}
+
+resource "pagerduty_escalation_policy" "foo" {
+  name      = "Engineering Escalation Policy"
+  num_loops = 2
+
+  rule {
+    escalation_delay_in_minutes = 10
+
+    target {
+      type = "user"
+      id   = pagerduty_user.example.id
+    }
+  }
+}
+
+resource "pagerduty_service" "example" {
+  name                    = "My Web App"
+  auto_resolve_timeout    = 14400
+  acknowledgement_timeout = 600
+  escalation_policy       = pagerduty_escalation_policy.example.id
```

Comment:
Since we're making other edits to this doc. This should probably match the definition above for `pagerduty_escalation_policy.foo.id`.